packages/drag-in-the-blank/controller/src/utils.js

```diff
@@ -24,4 +24,14 @@ export const getAllCorrectResponses = ({ correctResponse, alternateResponses })
     possibleResponses: {},
     numberOfPossibleResponses: undefined
   });
-};
\ No newline at end of file
+};
+
+export const choiceIsEmpty = choice => {
+  if (choice) {
+    const { value = '' } = choice;
+
+    return value.trim() === '' || value.replace(/<[^>]*>?/gm, '') === '';
```

Comment:
I would move regex outside so it doesn't get created every single time the function runs. Regex instantiation costs quite a bit of memory.

Reply:
There's another one in `markupUtils.js`, same applies to that one as well.